pom.xml

```diff
@@ -169,9 +169,9 @@
       <url>http://spoon.gforge.inria.fr/repositories/releases</url>
     </repository>
     <repository>
-      <id>gforge.inria.fr-snapshot</id>
+      <id>maven.inria.fr-snapshot</id>
```

Comment:
I guess we can remove the whole `repository` tag since we release on Central, and lonly keep the `snapshotRepository`?

Reply:
I suppose we can indeed. I just kept the same pattern as before.